Source/Parallelization/WarpXComm.cpp

```diff
@@ -863,6 +863,8 @@ WarpX::SyncCurrent (
     // If warpx.do_current_centering = 1, center currents from nodal grid to staggered grid
     if (WarpX::do_current_centering)
     {
+        AMREX_ALWAYS_ASSERT_WITH_MESSAGE(finest_level <= 1,
+                                         "warpx.do_current_centering=1 not supported with more than one fine levels");
```

Comment:
To be safe, I added an assertion here, because I am not sure if we need to do anything special for do_current_centering.

Reply:
The assertion is a good idea for now, I think current centering is not fully supported with mesh refinement yet.